docs/examples/Creating_Actionable_Sensor.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+Creating an Actionable Sensor Example
```

Comment:
<img width="751" alt="Screenshot 2021-11-18 at 14 24 19" src="https://user-images.githubusercontent.com/31851985/142433638-b5bfefb6-8e97-4c72-ac2a-48058340ea1a.png">

Reply:
Odd. I can't see why that's being raised.

Reply:
That's weird - it seems to load fine for me!